src/biogeophys/CanopyHydrologyMod.F90

```diff
@@ -294,8 +327,8 @@ subroutine CanopyHydrology(bounds, &
                       fpi = 0.25_r8*(1._r8 - exp(-0.5_r8*(elai(p) + esai(p))))
                    endif
 
-                   snocanmx = 60._r8*dewmx(p) * (elai(p) + esai(p))  ! 6*(LAI+SAI)
-                   liqcanmx = dewmx(p) * (elai(p) + esai(p))
+                   snocanmx = params_inst%sno_stor_max * (elai(p) + esai(p))  ! 6*(LAI+SAI)
+                   liqcanmx = params_inst%dewmx * (elai(p) + esai(p))
 
```

Comment:
Hi @swensosc @olyson @dlawrenncar , this is a question about two lines of code in CanopyHydrologyMod. If none of you know the answer, pls add the right person to this conversation:

In this PR I changed snocanmx from varying by dewmx (=0.1) to varying by sno_stor_max (= 6).

When calculating fcansno further down, I replaced the 6 with sno_stor_max without removing dewmx from the calculation. Bill correctly suggests that this has the potential of introducing an inconsistency if we modify the values of dewmx and/or sno_stor_max.

The question is... Which is the preferred formula for snocanmx? The one that varies by sno_stor_max OR the one that varies by both dewmx and sno_stor_max?